frontend/src/pages/signup/Signup.tsx

```diff
@@ -76,7 +71,7 @@ export default function SignUpPage() {
                     variant='outlined'
                     type='text'
                     value={firstName}
-                    onChange={(e) => setFirstName(e.target.value)}
+                    onChange={(e: any) => setFirstName(e.target.value)}
```

Comment:
e is `ChangeEvent<HTMLTextAreaElement>` type, you can import `ChangeEvent` from React library directly.

you can check the prop's type when you hover over it. It will tell you its type

Reply:
Actually, just remove the type `any` it should use the correct type by default. Thanks!